src/ILVerify/src/ILImporter.Verify.cs

```diff
@@ -476,6 +476,16 @@ void StartImportingBasicBlock(BasicBlock basicBlock)
                     break;
                 }
             }
+
+            if (basicBlock?.EntryStack?.Length > 0)
```

Comment:
basicBlock should be non-null here. Can this be just `basicBlock.EntryStack?.Length` ?